### ISSUE TITLE
Installation errors on Windows XP with SharpKeys 3.9.4

### ISSUE THREAD

**Issue by Seedy-Threepio** [randyrants/sharpkeys#447]
Hi
When I try to run the MSI installer on Windows XP, it says this installation package is not supported by this processor type. You say the MSI works on both X64 and X86, however this doesn't seem to be the case; it seems to only want to run on X64.
The ZIP doesn't work either. When I extract the zip file and run the executable, it says sharpkeys.exe is not a valid win32 application.
The machine is running Windows XP SP 3, and has DotNet Framework 4.0 installed as per the system requirements.

**Comment by randyrants** [comment]
Hate to say it, but your best bet is to try an older build of the app.  I compile the source with "Any CPU" which is means it should run on x86, x64, ARM, or ARM64 without any error - I've tested this at different times over the years and I did just move to Visual Studio 2022 so maybe there's something "new" being set in the compiler that prevents an OS from 2001 from running it.  Can't do anything for that.

I will look into the MSI and make sure that I didn't compile that with a "x64" only flag somewhere - building installers projects is always a crap shoot.
